src/__tests__/__mocks__/styleMock.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
What does this file do?

Reply:
it is needed to make the test pass because we will be using custom CSS and static files need to be handled, for further understanding, you can check this [Jest Doc](https://jestjs.io/docs/en/webpack)